easybuild/easyconfigs/k/KneadData/KneadData-0.6.1-foss-2017a-Python-2.7.13-Java-1.8.0_162.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'PythonPackage'
+
+name = 'KneadData'
+version = '0.6.1'
+versionsuffix = '-Python-%(pyver)s-Java-%(javaver)s'
+
+homepage = 'https://pypi.python.org/pypi/kneaddata'
+description = """KneadData is a tool designed to perform quality control on metagenomic
+ sequencing data, especially data from microbiome experiments."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['46540a22bebc84919a71138c419c69f68e9f5e35404587a2e31a3a3e53c66077']
+
+# KneadData does not itself use Java, but it has two dependencies that do, so
+# depending directly seems like an easy way to make sure that the versions
+# used by those deps are the same.
```

Comment:
There's no need to include a direct dep on `Java` for this purpose, I feel it's even confusing to do so.

If you include two dependencies that require a different version of `Java`, our test suite is going to notice the conflict and complain about it.

Also, you can use `eb --check-conflicts` to ensure that a dependency graph doesn't contain any conflicts.

Reply:
Attempted what I thought you were asking for here, but the new version doesn't resolve.  Not sure how to proceed.